dspy/streaming/streamify.py

```diff
@@ -193,6 +193,12 @@ async def async_streamer(*args, **kwargs):
                 elif isinstance(value, StatusMessage):
                     yield value
                 elif isinstance(value, Prediction):
+                    # Finalize all active listeners before yielding the prediction to flush any remaining buffered tokens
```

Comment:
nit: "Finalize" is a word defined by the code, let's just say "Flush remaining buffered tokens before yielding the `Prediction` instance"